react-app/src/App.js

```diff
@@ -24,9 +27,11 @@ function App() {
     return null;
   }
 
+
   return (
     <BrowserRouter>
-      <NavBar />
+      <ProfileButton user={testUser}/>
```

Comment:
The ProfileButton can just be added in the `NavBar`. Use a selector for the `user` from the `session` slice of the Redux store. Use conditional rendering to only display the button if the `user` is not `null`.